packages/components/inputs/rich-text-input/src/editor.tsx

```diff
@@ -56,23 +106,21 @@ const Editor = (props) => {
         ) {
           toggle();
         }
+        const refObj: TRichtTextEditorBodyRef = {
+          containerRef: ref,
+          registerContentNode,
+        };
         return (
           <Constraints.Horizontal max={props.horizontalConstraint}>
             <Stack scale="xs" alignItems="flex-end">
-              <EditorWrapper
-                isDisabled={props.isDisabled}
-                isReadOnly={props.isReadOnly}
-              >
+              <EditorWrapper isDisabled={props.isDisabled}>
                 <RichTextBody
-                  ref={{
-                    containerRef: ref,
-                    registerContentNode,
-                  }}
+                  ref={refObj as unknown as Ref<TRichtTextEditorBodyRef>}
                   hasError={props.hasError}
                   isDisabled={props.isDisabled}
                   hasWarning={props.hasWarning}
-                  isReadOnly={props.isReadOnly}
-                  showExpandIcon={props.showExpandIcon}
+                  isReadOnly={Boolean(props.isReadOnly)}
```

Comment:
I'm curious about these castings as both props are already defined as boolean.

Reply:
Thanks, I had to do that since both props are optional (so the type is `boolean | undefined`)

Reply:
Ah, ok! They are optional props of the `editor.tsx` but are mandatory props in the `RichTextBody`. Now I see.

Thanks.